docs/asset-store/docs/08-01-gateway-mode-set-up.md

```diff
@@ -0,0 +1,39 @@
+---
+title: Set Minio to the Google Cloud Storage (GCS) Gateway mode
+type: Tutorials
+---
+
+This tutorial shows how to set Minio to the Google Cloud Storage (GCS) Gateway mode. Follow these guidelines to create a secret and update the Asset Store deployment.
+
+## Prerequisites
+
+- [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/#install-kubectl)
+- [helm](https://github.com/helm/helm#install)
+- [Google Cloud Platform (GCP)](https://cloud.google.com)  access
+
+## Steps
+
+To set Minio to a Gateway mode, you need a Secret with a service account that has the **Storage Admin** role permissions.
+
+### Create a Secret
```

Comment:
I don't like UI instruction if stuff can be done with CLI
add gcloud to prerequisites and explan how to create a service account and access key as it is done in this instruction https://github.com/kyma-project/kyma/blob/master/docs/kyma/docs/04-04-gke-installation.md#get-the-tls-certificate